js/build/shortcode-ui.js

```diff
@@ -167,7 +167,15 @@ var Shortcode = Backbone.Model.extend({
 			if ( attr.get( 'attr' ) === 'content' ) {
 				content = attr.get( 'value' );
 			} else {
-				attrs.push( attr.get( 'attr' ) + '="' + attr.get( 'value' ) + '"' );
+				
+				// Numeric attribute names
+				if ( ! isNaN( attr.get( 'attr' ) ) ) {
```

Comment:
Lots of double negatives here... We could check `$.isNumeric( attr.get( 'attr' ) )` instead?